docs/framework_events.rst

```diff
@@ -4,6 +4,95 @@
 Custom Events
 =============
 
+Audio
+^^^^^
+
+.. py:method:: Red.on_red_audio_audio_disconnect(guild)
+
+    Dispatched when a player disconnects from a channel
+
+    :param discord.Guild guild: The guild in which the player disconnected
+
+.. py:method:: Red.on_red_audio_queue_end(guild, prev_track, prev_requester)
+
+    Dispatched when the queue ends
+
+    :param discord.Guild guild: The guild in which the queue ended
+    :param lavalink.Track prev_track: The last track
+    :param discord.Member prev_requester: The track's requester
+
+.. py:method:: Red.on_red_audio_skip_track(guild, prev_track, prev_requester)
+
+    Dispatched when a track is skipped
+
+    :param discord.Guild guild: The guild in which the track was skipped
+    :param lavalink.Track prev_track: The track that was skipped
+    :param discord.Member prev_requester: The track's requester
+
+.. py:method:: Red.on_red_audio_track_auto_play(guild, track, requester, player)
+
+    Dispatched when a track is auto-played
+
+    :param discord.Guild guild: The guild in which the track was auto-played
+    :param lavalink.Track track: The track that was started
+    :param discord.Member requester: The track's requester: :py:meth:`discord.Guild.me` in this case
+    :param lavalink.Player player: The player associated to this event
+
+.. py:method:: Red.on_red_audio_track_enqueue(guild, track, requester)
+
+    Dispatched when a track is enqueued
+
+    :param discord.Guild guild: The guild in which the track was enqueued
+    :param lavalink.Track track: The track that was enqueued
+    :param discord.Member requester: The track's requester
+
+.. py:method:: Red.on_red_audio_track_end(guild, track, requester)
+
+    Dispatched when a playing track ends
+
+    :param discord.Guild guild: The guild in which the track ended
+    :param lavalink.Track track: The track that ended
+    :param discord.Member requester: The track's requester
+
+.. py:method:: Red.on_red_audio_track_start(guild, track, requester)
+
+    Dispatched when an enqueued track starts playing
+
+    :param discord.Guild guild: The guild in which the track started
+    :param lavalink.Track track: The track that was started
+    :param discord.Member requester: The track's requester
+
+.. py:method:: Red.on_red_audio_unload(cog)
+
+    Dispatched when the audio cog starts it's unloading task
+
+    :param cog: Audio cog instance: similar to :py:func:`redbot.core.bot.RedBase.get_cog()`
```

Comment:
@aikaterna do we actually support these?

Reply:
Thanks for having a look Jack. Sorry for not answering for such a long time. Christmas holidays.

Regarding your question for aika: I cannot really tell you whether the audio events are supported or not, they seem to work just fine tho. I did some basic testing with all of them and they seem to be dispatched reliably.

Reply:
They have been in audio since 2018 or so I believe but they were never documented (obviously). They have been there for people to make use of and I don't know of any issues anyone has ever had with them. I would say they are supported imo.